src/api/custom-resolvers/mutations/create-many-applications/index.ts

```diff
@@ -0,0 +1,101 @@
+import { CreateManyApplicationsVariables, Application, Context, ObjectType } from '../../../types';
+import getCountries from '../../../helpers/get-countries';
+import { mockApplicationCreationObject } from '../../../test-mocks/mock-application';
+
+/**
+ * createManyApplications
+ * Creates many applications.
+ * This is used by E2E tests only
+ * 1) Creates an empty object array with the number of elements passed in variables
+ * 2) Gets all countries and creates a buyer with the first country
+ * 3) Maps application creation object
+ * 4) Creates many applications
+ * 5) Creates reference numbers for applications and updated ReferenceNumber and Application tables
+ * 6) returns applications array and success flag
+ * @param {Object} root: GraphQL root variables
+ * @param {CreateManyApplicationsVariables} GraphQL variables for the createManyApplications mutation
+ * @param {Context} context: KeystoneJS context API
+ * @returns {Promise<Object>} Object with success flag and applications array
+ */
+const createManyApplications = async (root: any, variables: CreateManyApplicationsVariables, context: Context) => {
+  console.info('Creating many applications');
+
+  try {
+    const emptyArray = new Array(variables.count).fill({});
+
+    const countries = await getCountries(context);
+
+    // creates buyer with first country from countries array
+    const buyer = await context.db.Buyer.createOne({
+      data: {
+        country: {
+          connect: {
+            id: countries[0].id,
+          },
+        },
+      },
+    });
```

Comment:
Could we move this into a helper, `createABuyer`, that accepts a country ID?

We already have an existing `createABuyer` helper, but it's fully populated. I think we should rename this to `createAPopulatedBuyer`